beta-src/src/webDiplomacyTheme.ts

```diff
@@ -86,6 +86,12 @@ const webDiplomacyTheme = createTheme({
         root: {
           borderRadius: 18,
           padding: "10px 18px",
+          "&.WDScrollButton, &.WDScrollButton:hover, &.WDScrollButton:focus": {
```

Comment:
let's not add this to the theme...you can use sx prop on Button to override the styles instead.